File: python/penisparser.py
#!usr/bin/python

def penis():
    filename = raw_input("file name?: ")
    penis = ""

    with open(filename) as f:
        for line in f:
            for word in line.split():
                if len(word) <= len(penis):
                    penis = "penis"
                else:
                    length = len(word) - 5
                    penis = "pe"
                    for penis in length:
                        penis = penis + "e"
                    penis = penis + "nis"
                
                print line.replace(str(word, penis) 
penis()

File: perl/penisparser.pl
#!/usr/bin/perl

use strict;

my $file = shift;

if (not $file)
{
    print "Usage: $0 file\n";
    exit 1;
}

open my $f, '<', $file or die "Can not open $file: $!\n";

undef $/;
(my $content = <$f>) =~ s/\w+/penis/g;
print $content;
